refextract/references/regexs.py

```diff
@@ -663,7 +663,7 @@ def compute_months():
     (\.\w+)*                                # subdivisions separated by . (doesn't have to be present)
     (/|%2f)                                 # / (possibly urlencoded)
     [\w\-_:;\(\)/\.<>]+                     # any character
-    [\w\-_:;\(\)/<>])                       # any character excluding a full stop
+    [\w\-_:\(\)<>])                       # any character excluding ; / or a full stop
```

Comment:
so this partially addresses the case of a line break pulling a DOI apart at `\` or `;`  
however it doesn't make any attempt to stitch the DOI together
so what's the benefit of dropping that last character -- which would make it more obvious that the DOI was truncated?